packages/astro/src/types/public/config.ts

```diff
@@ -112,7 +142,10 @@ export interface ViteUserConfig extends OriginalViteUserConfig {
  * Docs: https://docs.astro.build/reference/configuration-reference/
  *
  * Generics do not follow semver and may change at any time.
- */ export interface AstroUserConfig<TLocales extends Locales = never> {
+ */ export interface AstroUserConfig<
+	TLocales extends Locales = never,
+	TSession extends SessionDriverName = never,
```

Comment:
I'm glad the generic `defineConfig` is used already :)

Reply:
Yeah, it works great!